public/app/features/provisioning/dashboard.ts

```diff
@@ -18,7 +18,8 @@ export async function loadDashboardFromProvisioning(repo: string, path: string):
     .get(url, ref ? { ref } : undefined)
     .then((v) => {
       // Load the results from dryRun
-      const dryRun = v.resource.dryRun;
+      // TODO Some dashboards have a file resource, others have a dashboard resource :shrug:
+      const dryRun = v.resource.dryRun || v.resource.file?.resource?.dryRun;
```

Comment:
```suggestion
      const dryRun = v.resource.dryRun;
```
the second is an error in the file that is saved!